web/ui/react-app/src/pages/targets/ToggleMoreLess.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { Button } from 'reactstrap';
+
+interface ToggleProps {
+  onClick(): void;
+}
+
+export const ToggleMoreLess: FC<ToggleProps> = ({ children, onClick }) => {
+  const [showMoreLess, toggleMoreLess] = useState(false);
```

Comment:
As noted above, better to remove the state inside the component and only pass in an `expanded` prop from the outside which can be controlled from the button via the `onClick` function that is already passed in.